services/121-service/swagger.json

```diff
@@ -200,7 +200,16 @@
   },
   {
     "method": "get",
-    "path": "/api/programs/{programId}/fsps/intersolve-voucher/vouchers",
+    "path": "/api/programs/{programId}/fsps/intersolve-voucher/vouchers-paper",
```

Comment:
please create a documentation task where you follow the new documentation page in our internal documentation to check what needs updating (at least the API-model)